docs/cnv-configuration.md

```diff
@@ -0,0 +1,153 @@
+
+For both limits and requests with the `/vmi` suffix, the same values will apply because, unlike pods, virtual
+machines are already limited to the virtual memory and virtual CPU that were allocated to them.
+
+### Application Aware Resource Quota
+The ApplicationAwareResourceQuota API is compatible with the native ResourceQuota, meaning it shares the same specification and status definitions.
+
+Example manifest:
+
+```yaml
+apiVersion: aaq.kubevirt.io/v1alpha1
+kind: ApplicationAwareResourceQuota
+metadata:
+  name: example-resource-quota
+spec:
+  hard:
+    requests.memory: 1Gi
+    limits.memory: 1Gi
+    requests.cpu/vmi: "1"
+    requests.memory/vmi: 1Gi
+```
+
+### Application Aware Cluster Resource Quota
+This quota is API compatible with ClusterResourceQuota, meaning it shares the same specification and status definitions.
+
+The main differences between the AAQ cluster quota and AAQ quota are:
+
+- unlike AAQ quotas AAQ cluster quotas are not namespaced.
+- When creating AAQ cluster quota, you can select multiple namespaces based
+  on annotation selection, label selection, or both though`spec.selector.labels` or
+  `spec.selector.annotations`. These fields follows the standard selector format.
+
+
+  **Note**: If both `spec.selector.labels` and`spec.selector.annotations` are set only namespaces
+  that match both will be selected
+
+  **Note**: AAQ must be configured to allow AAQ cluster quota management
+
+  Refer to the "Configure AAQ Operator" section for more details. 
+
+Example manifest:
+```yaml
+apiVersion: aaq.kubevirt.io/v1alpha1
+kind: ApplicationAwareClusterResourceQuota
+metadata:
+  name: example-resource-quota
+spec:
+  quota:
+    hard:
+    requests.memory: 1Gi
+    limits.memory: 1Gi
+    requests.cpu/vmi: "1"
+    requests.memory/vmi: 1Gi
+  selector:
+    annotations: null
+    labels:
+      matchLabels:
+        kubernetes.io/metadata.name: default
+```
+
+### How it works
+
+These new quotas are controller-based rather than admission-based. Here’s how they work:
+
+1. [Scheduling Gates](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness/): a scheduling gates is added to    the created pod via a mutating webhook.
+   See the "Configure AAQ Operator" section below for more information
+2. Controller Evaluation: a controller evaluates whether
+   there is sufficient room for the pod according to the quota.
+3. Quotas Update: If there is enough room, the scheduling gate is 
+   removed from the pod, and the quotas used status is updated accordingly.
+   If not, an event is created explaining why the quota doesn't allow the pod's creation.
+
+For non-compute resources, the quota acts like the standard ResourceQuota.
+
+***Warning***: Pods that set `.spec.nodeName` are not allowed to use in any namespace where AAQ Operates.
+
+non-schedulable resources are supported by both AAQ quota and AAQ cluster 
+quota by managing a k8s resource quota behind the scenes.
```

Comment:
Does it mean that AAQ opeator creates ResourceQuota and ClusterResourceQouta?

Reply:
Yes, the AAQ operator creates ResourceQuota and ClusterResourceQuota to enforce limits on non-schedulable resources like PVCs or ClusterRoles. Since these can’t be managed with scheduling gates, AAQ uses native Kubernetes quotas as an alternative. This way, users don’t need to manually create separate native quotas for non-schedulable resources and AAQ quotas for pods.